orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/job/WaitOnJobCompletion.groovy

```diff
@@ -48,20 +49,12 @@ public class WaitOnJobCompletion implements CloudProviderAware, OverridableTimeo
   final long backoffPeriod = TimeUnit.SECONDS.toMillis(10)
   final long timeout = TimeUnit.MINUTES.toMillis(120)
 
-  @Autowired
-  KatoRestService katoRestService
-
-  @Autowired
-  ObjectMapper objectMapper
-
-  @Autowired
-  RetrySupport retrySupport
-
-  @Autowired
-  JobUtils jobUtils
-
-  @Autowired(required = false)
+  private final KatoRestService katoRestService
+  private final ObjectMapper objectMapper
+  private final RetrySupport retrySupport
+  private final JobUtils jobUtils
   Front50Service front50Service
+  private final ExecutionRepository repository
```

Comment:
Minor stylistic thing, can we keep final properties together?

Reply:
sure thing